config/nunjuck.js

```diff
@@ -1,6 +1,8 @@
 'use strict';
 
 const path = require('path');
+require('dotenv').config();
```

Comment:
Add the `require('dotenv').config()` before the `const path = require('path')`. This won't change much here, but it's good to keep the pattern of, when needed, running `dotenv` before anything else.